gpMgmt/bin/gpexpand

```diff
@@ -773,7 +773,18 @@ class SegmentTemplate:
 
     def _distribute_tarfile(self):
         """Distributes template tar file to hosts"""
+        localHostname = getLocalHostname()
+        # NOTE: Currently, GP's scp transfers relative file paths into the home
+        # directory by default.
+        pathsAreSame = (os.path.realpath(self.segTarFile) == os.path.join('/home/gpadmin/', self.schema_tar_file))
```

Comment:
```suggestion
        pathsAreSame = os.path.realpath(self.segTarFile) == os.path.join('/home/gpadmin', self.schema_tar_file)
```

Reply:
If user is not gpadmin, it will not work. Use `getUserName()`.

Reply:
> I don't think this makes any difference :c

Extra parentheses are not welcome in Python.